lightning-background-processor/src/lib.rs

```diff
@@ -161,8 +146,8 @@ impl BackgroundProcessor {
 	/// [`stop`]: Self::stop
 	/// [`ChannelManager`]: lightning::ln::channelmanager::ChannelManager
 	/// [`ChannelManager::write`]: lightning::ln::channelmanager::ChannelManager#impl-Writeable
-	/// [`FilesystemPersister::persist_manager`]: lightning_persister::FilesystemPersister::persist_manager
-	/// [`FilesystemPersister::persist_network_graph`]: lightning_persister::FilesystemPersister::persist_network_graph
+	/// [`FilesystemPersister::persist_manager`]: lightning_persister::FilesystemPersister#impl-Persister
```

Comment:
Hmm, this link doesn't work now, it seems. I think maybe just link directly to `Persister::persist_*`